README.md

```diff
@@ -8,9 +8,112 @@ This library works with both plain JS as well as AngularJS applications.
 
 [![Build Status](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-js.svg?branch=dev)](https://travis-ci.org/AzureAD/azure-activedirectory-library-for-js)[![npm](https://img.shields.io/npm/v/adal-angular.svg)](https://www.npmjs.com/package/adal-angular)[![npm](https://img.shields.io/npm/dm/adal-angular.svg)](https://www.npmjs.com/package/adal-angular)
 
-##  Usage
 
-You can learn in detail about ADAL JS installation and usage documented in the [ADAL Basics Wiki](https://github.com/AzureAD/azure-activedirectory-library-for-js/wiki/ADAL-basics).
+## Installation
+
+You have multiple ways of getting ADAL JS:
+
+Via NPM (https://www.npmjs.com/package/adal-angular):
+
+    npm install adal-angular
+
+*Note:* Currently there is one NPM package providing both the plain JS library (adal.js) and the AngularJS wrapper (adal-angular.js).
+
+Via CDN:
+
+    <!-- Latest compiled and minified JavaScript -->
+    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.17/js/adal.min.js"></script>
+    <script src="https://secure.aadcdn.microsoftonline-p.com/lib/1.0.17/js/adal-angular.min.js"></script>
+
+Via Bower:
+
+    $ bower install adal-angular
+
+## Usage
+
+#### Prerequisite
+
+Before using ADAL JS, follow the instructions to [register your application](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-integrating-applications) on the Azure portal. Also, make sure to enable the [OAuth 2.0 implicit flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v1-oauth2-implicit-grant-flow) by setting the property `oauth2AllowImplicitFlow` to true by editing your application manifest on the portal. Implicit flow is used by ADAL JS to get tokens.
+
+#### 1. Instantiate the AuthenticationContext
+
+Instantiate the global variable AuthenticationContext with a minimal required config of clientID. You can read about other configurable options [here](https://github.com/AzureAD/azure-activedirectory-library-for-js/wiki/Config-authentication-context#configurable-options-).
```

Comment:
can we remove the - at the end of https://github.com/AzureAD/azure-activedirectory-library-for-js/wiki/Config-authentication-context#configurable-options ?